.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -84,6 +84,7 @@ stages:
               os: linux

```

Comment:
Are we not adding the logs Nightly pipeline for cilium ?

Reply:
The nightly pipeline calls in the cilium-overlay template, but I should add a job that captures the failures! Good catch.